include/ishikawa.hrl

```diff
@@ -1,3 +1,10 @@
 -type actor() :: binary().
 -type message() :: binary().
 -type timestamp() :: vclock:vclock().
+-type timestampMatrix() :: mclock:mclock().
+-record(state, {actor :: actor(),
+                vv :: timestamp(),
+                members :: [term()],
+                svv :: timestamp(),
+                rtm :: timestampMatrix(),
+                toBeDlvrdQ :: [{timestamp(), message()}]}).
```

Comment:
It would be useful to switch this to snake_case and be more explicit about the naming -- I don't easily know what this is.